packages/typespec-ts/test/integration/generated/overload/src/clientDefinitions.ts

```diff
@@ -2,58 +2,41 @@
 // Licensed under the MIT license.
 
 import {
-  GetStringParameters,
-  GetNumberParameters,
-  UploadStringParameters,
-  UploadBytesParameters,
-  ProcessStringParameters,
-  ProcessBytesParameters,
+  GetThingParameters,
```

Comment:
The same for the overload test case. https://github.com/Azure/autorest.typescript/blob/main/packages/typespec-ts/test/integration/typespec/overload/client.tsp